### ISSUE TITLE
Hotstar.com IPL livestream

### ISSUE THREAD

**Issue by brpy** [ytdl-org/youtube-dl#28714]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

The ipl live stream on hotstar.com does not look to be using DRM and I want to stream it using mpv using my account.

I've tried passing cookies but I get `ERROR: This video is only available for registered users. Use --username and --password or --netrc to provide account credentials.`. But hotstar doesn't use a userid , password combo. It uses phone number + otp(one time password). Any help?

Thanks.

**Comment by Guri99** [comment]
I do not think that you will be able to integrate it with hotstar because as you said it uses OTP.

**Comment by 89z** [comment]
Issue should be closed, no example URL
